src/Api/Droplet.php

```diff
@@ -163,7 +163,7 @@ public function create($names, $region, $size, $image, $backups = false, $ipv6 =
         $droplet = json_decode($droplet);
 
         if (is_array($names)) {
-            return array_map(function ($droplet) {
+            return array_map(function ($droplet) use ($wait, $waitTimeout) {
```

Comment:
Seems like creating more than one droplet would have ignored the wait parameters?